webserver/http2/src/main/java/io/helidon/webserver/http2/Http2ServerStream.java

```diff
@@ -114,7 +116,8 @@ public class Http2ServerStream implements Runnable, Http2Stream {
      * @param writer                writer
      * @param connectionFlowControl connection flow control
      */
-    public Http2ServerStream(ConnectionContext ctx,
+    Http2ServerStream(ConnectionContext ctx,
```

Comment:
This deserves reformatting of parameters, as the `public` was removed

Reply:
fixed